eng/Versions.props

```diff
@@ -9,19 +9,19 @@
     <!-- Packages from dotnet/arcade -->
     <MicrosoftDotNetBuildTasksTemplatingVersion>6.0.0-beta.21373.11</MicrosoftDotNetBuildTasksTemplatingVersion>
     <!-- Packages from dotnet/aspnetcore -->
-    <MicrosoftAspNetCoreApiAuthorizationIdentityServerVersion>6.0.0-preview.6.21355.2</MicrosoftAspNetCoreApiAuthorizationIdentityServerVersion>
-    <MicrosoftAspNetCoreDiagnosticsEntityFrameworkCoreVersion>6.0.0-preview.6.21355.2</MicrosoftAspNetCoreDiagnosticsEntityFrameworkCoreVersion>
-    <MicrosoftAspNetCoreIdentityEntityFrameworkCoreVersion>6.0.0-preview.6.21355.2</MicrosoftAspNetCoreIdentityEntityFrameworkCoreVersion>
-    <MicrosoftAspNetCoreIdentityUIVersion>6.0.0-preview.6.21355.2</MicrosoftAspNetCoreIdentityUIVersion>
-    <MicrosoftAspNetCoreSpaProxyVersion>6.0.0-preview.6.21355.2</MicrosoftAspNetCoreSpaProxyVersion>
-    <MicrosoftAspNetCoreSpaServicesExtensionsVersion>6.0.0-preview.6.21355.2</MicrosoftAspNetCoreSpaServicesExtensionsVersion>
+    <MicrosoftAspNetCoreApiAuthorizationIdentityServerVersion>6.0.0</MicrosoftAspNetCoreApiAuthorizationIdentityServerVersion>
```

Comment:
Out of curiosity, how does this work in the template package? Do these versions get stomped over when it's consumed as a submodule in aspnetcore?

Reply:
Yeah, these are only for local development

Reply:
Should we consider using the patched version `6.0.1`?

Reply:
Nah, these are only used here for local development. When built as part of the submodule they get the versions from there